tap_pardot/streams.py

```diff
@@ -60,7 +60,16 @@ def get_records(self):
         records = data["result"][self.data_key]
         if isinstance(records, dict):
             records = [records]
-        return records
+
+        return map(self.flatten_record, records)
+
+    def flatten_record(self, record):
+        """ In case when data comes as a dict with 'value' key only.
+        """
+        for key, value in record.items():
+            if isinstance(value, dict) and 'value' in value:
+                record[key] = value['value']
+        return record
```

Comment:
I'm guessing we're only seeing values objects at the top-level? Otherwise, we'd have to do this recursively. If that is the case, this is the right approach.

Reply:
Yes, only top level